packages/backend/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Might want to add this into the `*.code-workspace` files and the root level `.vscode/settings.json`.

Reply:
Thank you, did that in homepage branch.